.github/workflows/ci.yml

```diff
@@ -16,30 +19,31 @@ jobs:
   build:
     name: Check and build codebase
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [16.x, 18.x]
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 18.x
 
       # Wireit cache
       - uses: google/wireit@setup-github-actions-caching/v1
 
-      - uses: pnpm/action-setup@v2.2.2
+      - uses: pnpm/action-setup@v3
+        name: Install pnpm
         with:
+          run_install: false
           version: 7
 
       - name: Get pnpm store directory
+        shell: bash
         id: pnpm-cache
-        run: echo "::set-output name=pnpm_cache_dir::$(pnpm store path)"
+        run: |
+          echo "PNPM_STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
```

Comment:
this clears the warning outlined [here](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) and in the pipeline logs

Reply:
Oh good call. I'm surprised to not see this mentioned anywhere in the pnpm/action-setup documentation. It still lists out the manual strategy for caching.

Reply:
@carbonrobot This has been completed. Pipeline logs indicate the cache directory was found 👍🏻 
![Uploading image.png…]()